olmbindata/olm_versions.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olmbindata
+
+func GetAvailableOLMVersions() map[string]bool {
+	return map[string]bool{
+		"0.15.1": true,
+	}
+}
```

Comment:
Right now we only need to look up if a version is available:
```suggestion
var availableVersions = map[string]struct{}{
	"0.15.1": struct{}{},
}

func HasVersion(version string) bool {
	_, ok := availableVersions[version]
	return ok
}
	
```

Reply:
Would it be fine to have this in the same package as olm_bindata so that we need not update client.go when an additional olm version included?

Reply:
It should be in the same package as bindata.